packages/preview/unibuc-fmi-thesis-template/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "unibuc-fmi-thesis-template"
```

Comment:
The name is a bit too canonical for what the package does. Can you please change it to something more unique? You can just add an adjective in front for instance.